worlds/tunic/__init__.py

```diff
@@ -0,0 +1,176 @@
+from typing import Dict, List, Any
+
+from BaseClasses import Region, Location, Item, Tutorial, ItemClassification
+from .Items import item_name_to_id, item_table, item_name_groups, fool_tiers, filler_items, slot_data_item_names
+from .Locations import location_table, location_name_groups, location_name_to_id, hexagon_locations
+from .Rules import set_location_rules, set_region_rules, randomize_ability_unlocks, gold_hexagon
+from .Regions import tunic_regions
+from .Options import tunic_options
+from worlds.AutoWorld import WebWorld, World
+from decimal import Decimal, ROUND_HALF_UP
+
+
+class TunicWeb(WebWorld):
+    tutorials = [
+        Tutorial(
+            tutorial_name="Multiworld Setup Guide",
+            description="A guide to setting up the TUNIC Randomizer for Archipelago multiworld games.",
+            language="English",
+            file_name="setup_en.md",
+            link="guide/en",
```

Comment:
These need to match. Right now, running WebHost.py on this branch produces a 404 for the tunic setup guide

Either rename the file to `guide_en.md` or change the link to `setup/en`

As far as I can tell, the `file_name` field here isn't actually used, as all docs get copied, and misc.py/tutorial.js effectively just replace the `/` in the URL link with a `_` to find the .md file

Reply:
Good catch, thanks! Should be fixed now.